cryptofeed/exchanges/mixins/bitfinex_rest.py

```diff
@@ -12,31 +12,43 @@
 
 from yapic import json
 
-from cryptofeed.defines import BID, ASK, BUY, L2_BOOK, L3_BOOK, SELL, TICKER, TRADES
```

Comment:
please undo this. I know many people prefer multiple lines for long lines, but that is not the formatting used in this project